concurrency-benchmarks.cabal

```diff
@@ -54,7 +79,11 @@ benchmark benchmarks
     transformers        >= 0.4   && < 0.6,
     async               >= 2.1.1 && < 2.3,
     random              >= 1.0   && < 2,
-    streamly            >= 0.4.0 && < 0.6
+    -- XXX This should ideally be >= 0.8.0 && < 0.9
```

Comment:
Let's keep it at 0.7.2 until we release 0.8.0.

Reply:
Made it `>= 0.7.2 && < 0.8.0` for the time being.